db/food_venue/food_venue.go

```diff
@@ -6,29 +6,33 @@ import (
 	"github.com/BurdockBH/food-delivery-rest-service/db"
 	"github.com/BurdockBH/food-delivery-rest-service/viewmodels"
 	"log"
-	"time"
 )
 
-func CreateFoodVenue(fv *viewmodels.FoodVenue) error {
-	query := "CALL CreateFoodVenue(?, ?, ?, ?)"
+func CreateFoodVenue(fv *viewmodels.FoodVenue, email string) error {
```

Comment:
Is created_by strictly an email or can it be a name? If it's an email are you validating it anywhere to see if it's in a correct format?

Reply:
The email is being validated in login user. The reason why it is not being validated, it's because it is derived from claims in token. When logging in, after validating email, user gets token which has to authorize venue creation. This email is not included in JSON that client is sending to backend.